appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -272,6 +280,12 @@ static ControllerLinkBuilder getAddUserLinkBuilder()
 		return linkTo(methodOn.addUser(Optional.empty(), null, null, null));
 	}
 
+	static ControllerLinkBuilder getUpdateUserLinkBuilder(UUID userId)
+	{
+		UserController methodOn = methodOn(UserController.class);
+		return linkTo(methodOn.updateUser(Optional.empty(), null, userId, null, null));
```

Comment:
I forgot we're using the IP address too. With that, I'm not very concerned. One hour would be a reasonable window. If someone ever legitimately runs into, it, they don't need to wait very long for it to expire.